src/Tes.SDK/TesClient.cs

```diff
@@ -40,7 +39,7 @@ private static async ValueTask<T> DeserializeAsync<T>(HttpContent content, Cance
         }
 
         /// <inheritdoc/>
-        public string SdkVersion { get; } = "0.1.0";
+        public string SdkVersion { get; } = "0.1.1";
```

Comment:
Why 0.1.1 and not 1.1.0 (which is the way it's defined @ https://github.com/ga4gh/task-execution-schemas/blob/1df37d34242f74d3f03475c6b9de3324b8094054/openapi/task_execution_service.openapi.yaml#L4